### ISSUE TITLE
How to download pretrained model?  bucket_namespace

### ISSUE THREAD

**Issue by LPXTT** [yyliu01/PS-MT#5]
Hello author, could you provide the download link of ResNet50 and ResNet101? I don't know the bucket_namespace and bucket_name in the download script. I downloaded two pretrained models from other place but the size dosen't match. Thanks.

**Comment by LPXTT** [comment]
Besides, I am new to semantic segmentation task and don't know how to preprocess cityscape data. Could you please give me some instructions?

**Comment by yyliu01** [comment]
Thanks for your interest.

The download link for the pre-trained model is same as the CPS, and you can find them in [here](https://onedrive.live.com/?authkey=%21AGEiz96zF%5FRougc&id=B71317D47B7AC1CB%21895&cid=B71317D47B7AC1CB). Please don't worry about those bucket_namespaces, they are for k8s & dgx-clustering launch. You can train our approach in the normal way.

The processed cityscapes can be found in this [repository](https://github.com/LiheYoung/ST-PlusPlus#readme), and please note that the directory for the cityscapes follow this structure:
```shell
city_scape/
├── annotation
│   └── city_gt_fine
│       ├── train
│       └── val
└── images
    └── city_gt_fine
        ├── train
        └── val
```

Best,
Yuyuan

**Comment by LPXTT** [comment]
Thank you!

发自我的iPhone

**Comment by LPXTT** [comment]
Sorry. I didn't find the processed cityscapes in this [repository](https://github.com/LiheYoung/ST-PlusPlus#readme). There are only original files not processed files.

**Comment by LPXTT** [comment]
Hi Yuyuan,
How can I get the SegmentationClassAug? I found SegmentationClass and SegmentationClassAug in vocsplit/*.txt. While, there is no SegmentationClassAug in pascalVOC.

**Comment by yyliu01** [comment]
@LPXTT The related download address can be found in this repository (here)[https://github.com/yassouali/CCT] for the SegmentationClassAug. For a brief explaination, the Pascal VOC 12 contains around ~1400 official (high-quality) labels and ~9000 unofficial low-quality labels. The SegmentationClassAug should contain those low-quality labels.

**Comment by LPXTT** [comment]
Thank you!
